### PR TITLE
update document of tidb-ctl

### PR DESCRIPTION
**Pull request by imtbkcat**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3189
- Other reference link(s):

### CODE REVIEW

**Comment by yikeke** [comment]
Does this PR need to be cherry-picked to release-2.0? @imtbkcat

**Comment by imtbkcat** [comment]
@yikeke no, it doesn't, some function in release-2.0 is different.

**Comment by yikeke** [comment]
> @yikeke no, it doesn't, some function in release-2.0 is different.

Sorry, I made a typo. I mean 2.1 (facepalm

File: tidb-control.md [hunk 1]
```diff
@@ -32,15 +40,33 @@ Usage example: `tidb-ctl schema in mysql -n db`
 - `-n`: the option
 - `db`: the flag of `-n`
 
+Currently, TiDB Control contains following commands:
```
Comment (TomShawn) [inline tidb-control.md:43]:
```suggestion
Currently, TiDB Control has the following subcommands:
```

File: tidb-control.md [hunk 2]
```diff
@@ -32,15 +40,33 @@ Usage example: `tidb-ctl schema in mysql -n db`
...
+
+- `tidb-ctl base64decode`: BASE64 decode
```
Comment (TomShawn) [inline tidb-control.md:45]:
```suggestion
- `tidb-ctl base64decode`: used for `BASE64` decoding
```

File: tidb-control.md [hunk 3]
```diff
@@ -32,15 +40,33 @@ Usage example: `tidb-ctl schema in mysql -n db`
...
+- `tidb-ctl decoder`: for KEY decode
```
Comment (TomShawn) [inline tidb-control.md:46]: language and format ditto

File: tidb-control.md [hunk 4]
```diff
@@ -32,15 +40,33 @@ Usage example: `tidb-ctl schema in mysql -n db`
...
+- `tidb-ctl etcd`: for operating etcd
+- `tidb-ctl log`: format the log file to expand the single-line stack information
```
Comment (TomShawn) [inline tidb-control.md:48]:
```suggestion
- `tidb-ctl log`: used to format the log file to expand the single-line stack information
```

File: tidb-control.md [hunk 5]
```diff
@@ -32,15 +40,33 @@ Usage example: `tidb-ctl schema in mysql -n db`
...
+- `tidb-ctl mvcc`: MVCC information
+- `tidb-ctl region`: Region information
+- `tidb-ctl schema`: Schema information
+- `tidb-ctl table`: Table information
```
Comment (TomShawn) [inline tidb-control.md:52]: used to get the ...

File: tidb-control.md [hunk 6]
```diff
@@ -32,15 +40,33 @@ Usage example: `tidb-ctl schema in mysql -n db`
...
 
-`tidb-ctl` has 4 connection related options:
+Use `tidb-ctl schema -h` to get usage details. The `schema` command itself has two subcommands: `in` and `tid`.
+
+- `in` is used to obtain the table schema of all tables in the database through the database name.
+- `tid` is used to obtain the table schema by using the unique `table_id` in the whole database.
+
+### Global options
+
+`tidb-ctl` has 7 connection related global options:
```
Comment (TomShawn) [inline tidb-control.md:69]:
```suggestion
`tidb-ctl` has the following connection-related global options:
```

File: tidb-control.md [hunk 7]
```diff
@@ -58,31 +84,20 @@ TiDB Control consists of multiple layers of commands. You can use `-h/--help` af
...
-- `tidb-ctl schema`: Schema information
-- `tidb-ctl table`: Table information
+- `tidb-ctl base64decode`
+- `tidb-ctl decoder`
+- `tidb-ctl etcd`
+- `tidb-ctl log`
+- `tidb-ctl mvcc`
+- `tidb-ctl region`
+- `tidb-ctl schema`
+- `tidb-ctl table`
```
Comment (TomShawn) [inline tidb-control.md:94]: I think you can remove this part.

File: tidb-control.md [hunk 8]
```diff
@@ -58,31 +84,20 @@ TiDB Control consists of multiple layers of commands. You can use `-h/--help` af
...
 
-For details about how to use the above `tidb-ctl` commands, use `tidb-ctl SUBCOMMAND --help` to get the help information.
+### The `schema` connand
```
Comment (TomShawn) [inline tidb-control.md:96]:
```suggestion
### The `schema` command
```

File: tidb-control.md [hunk 9]
```diff
@@ -58,31 +84,20 @@ TiDB Control consists of multiple layers of commands. You can use `-h/--help` af
...
-
-- `in` is used to obtain the table schema of all tables in the database through the database name.
-- `tid` is used to obtain the table schema by using the unique `table_id` in the whole database.
-
-#### The `in` command
-
-You can use `tidb-ctl schema in -h/--help` to get the help information of the `in` subcommand.
-
-##### Basic usage
+The `in` subcommand is used to get table information from target database.
```
Comment (TomShawn) [inline tidb-control.md:100]:
```suggestion
`in` is used to obtain the table schema of all tables in the database through the database name.
```

File: tidb-control.md [hunk 10]
```diff
@@ -128,6 +143,27 @@ The result is displayed in the JSON format. (The above output is truncated.)
 
     If you do not want to use the default TiDB service address and port, use the `--host` and `--port` options to configure. For example, `tidb-ctl --host 172.16.55.88 --port 8898 schema in mysql -n db`.
 
+#### The `tid` subcommand
+
+The `tid` subcommand is used to get table information and structure by table id.
```
Comment (TomShawn) [inline tidb-control.md:148]:
```suggestion
- `tid` is used to obtain the table schema by using the unique `table_id` in the whole database.
```

File: tidb-control.md [hunk 11]
```diff
@@ -128,6 +143,27 @@ The result is displayed in the JSON format. (The above output is truncated.)
...
+
+For example, the table id of `mysql.stat_meta` is 21, you can use `tidb-ctl schema tid -i 21` to obtain the detail of `mysql.stat_meta`.
```
Comment (TomShawn) [inline tidb-control.md:150]:
```suggestion
For example, the table ID of `mysql.stat_meta` is `21`. You can use `tidb-ctl schema tid -i 21` to obtain the detail of `mysql.stat_meta`.
```

File: tidb-control.md [hunk 12]
```diff
@@ -128,6 +143,27 @@ The result is displayed in the JSON format. (The above output is truncated.)
 
     If you do not want to use the default TiDB service address and port, use the `--host` and `--port` options to configure. For example, `tidb-ctl --host 172.16.55.88 --port 8898 schema in mysql -n db`.
 
+#### The `tid` subcommand
+
+The `tid` subcommand is used to get table information and structure by table id.
+
```
Comment (TomShawn) [inline tidb-control.md:149]: `通过使用 in 子命令查询到数据库中表的 id，之后可以通过 tid 子命令查看表的详细信息。` is missing from your translation.

File: tidb-control.md [hunk 13]
```diff
@@ -128,6 +143,27 @@ The result is displayed in the JSON format. (The above output is truncated.)
...
+  "O": "stats_meta",
+  "L": "stats_meta"
+ },
+ "charset": "utf8mb4",
+ "collate": "utf8mb4_bin",
+  ...
+}
+``` 
+
+Like the `in` subcommand, if you do not want to use the default TiDB service address and port, use the `--host` and `--port` options to configure.
```
Comment (TomShawn) [inline tidb-control.md:165]:
```suggestion
Like the `in` subcommand, if you do not want to use the default TiDB service address and port, use the `--host` and `--port` options to specify the host and port.
```

File: tidb-control.md [hunk 14]
```diff
@@ -128,6 +143,27 @@ The result is displayed in the JSON format. (The above output is truncated.)
...
+
 #### The `base64decode` subcommand
```
Comment (TomShawn) [inline tidb-control.md:167]:
```suggestion
#### The `base64decode` command
```

File: tidb-control.md [hunk 15]
```diff
@@ -7,11 +7,19 @@ aliases: ['/docs/dev/tidb-control/','/docs/dev/reference/tools/tidb-control/']
...
 # TiDB Control User Guide
 
-TiDB Control is a command line tool of TiDB, usually used to obtain the status information of TiDB for debugging.
+TiDB Control is a command-line tool of TiDB, usually used to obtain the status information of TiDB for debugging. This document introduces the features of TiDB Control and how to use these features.
 
-## Compile from source code
+## Get TiDB Control
 
-- Compilation environment requirement: [Go](https://golang.org/) Version 1.7 or later
+You can get TiDB Control using the following two methods.
```
Comment (TomShawn) [inline tidb-control.md:14]:
```suggestion
You can get TiDB Control by installing it using TiUP or by compiling it from source code.
```

File: tidb-control.md [hunk 16]
```diff
@@ -32,15 +40,33 @@ Usage example: `tidb-ctl schema in mysql -n db`
 - `-n`: the option
 - `db`: the flag of `-n`
 
+Currently, TiDB Control has the following subcommands:
+
+- `tidb-ctl base64decode`: used for `BASE64` decoding
+- `tidb-ctl decoder`: used for `KEY` decoding
+- `tidb-ctl etcd`: for operating etcd
+- `tidb-ctl log`: used to format the log file to expand the single-line stack information
+- `tidb-ctl mvcc`: used to get the mvcc information
```
Comment (TomShawn) [inline tidb-control.md:49]:
```suggestion
- `tidb-ctl mvcc`: used to get the MVCC information
```

File: tidb-control.md [hunk 17]
```diff
@@ -32,15 +40,33 @@ Usage example: `tidb-ctl schema in mysql -n db`
...
+- `tidb-ctl region`: used to get the region information
```
Comment (TomShawn) [inline tidb-control.md:50]:
```suggestion
- `tidb-ctl region`: used to get the Region information
```

File: tidb-control.md [hunk 18]
```diff
@@ -32,15 +40,33 @@ Usage example: `tidb-ctl schema in mysql -n db`
 - `-n`: the option
 - `db`: the flag of `-n`
 
+Currently, TiDB Control has the following subcommands:
+
+- `tidb-ctl base64decode`: used for `BASE64` decoding
+- `tidb-ctl decoder`: used for `KEY` decoding
+- `tidb-ctl etcd`: for operating etcd
```
Comment (TomShawn) [inline tidb-control.md:47]:
```suggestion
- `tidb-ctl etcd`: used for operating etcd
```

File: tidb-control.md [hunk 19]
```diff
@@ -128,7 +132,28 @@ The result is displayed in the JSON format. (The above output is truncated.)
 
     If you do not want to use the default TiDB service address and port, use the `--host` and `--port` options to configure. For example, `tidb-ctl --host 172.16.55.88 --port 8898 schema in mysql -n db`.
 
-#### The `base64decode` subcommand
+#### The `tid` subcommand
+
+- `tid` is used to obtain the table schema by using the unique `table_id` in the whole database. you can use `in` subcommand to get all table id of certain schema and use `tid` subcommand to get detail table information.
```
Comment (TomShawn) [inline tidb-control.md:137]:
```suggestion
`tid` is used to obtain the table schema by using the unique `table_id` in the whole database. You can use the `in` subcommand to get all table IDs of certain schema and use the `tid` subcommand to get the detailed table information.
```

**Pull request merged**